docs/guides/tutorials/prefarm-audit.md

```diff
@@ -0,0 +1,121 @@
+---
+slug: /guides/custody-tool-description
+title: Custody Tool Description
+---
+
+Chia Network Inc's prefarm is being secured by a complex set of custodial rules. This document will describe how to use the custody tool to audit the prefarm configuration. A moderate level of technical proficiency is probably needed to understand the details. For a high-level overview of the prefarm custody wallets, see our [blog post](https://www.chia.net/2022/10/29/a-new-home-for-the-prefarm/).
+
+Other relevant documents:
+
+- [Flow chart](/img/chia-custody-tool.png) to visualize how the custody tool works
+- [User guide](/guides/custody-tool-user-guide) to help you get up and running
+- [CLI reference](/custody-tool) for all custody commands used in this tutorial
+- [Prefarm Alert Tool](https://github.com/Chia-Network/prefarm-alert) for accessing the public prefarm config files
+
+## Prefarm Audit with the Internal Custody Tool
```

Comment:
Suggestion: Please consider using sentence case for H1-H3, and initial caps for titles.

Reply:
I used initial caps on H2 for alignment with other tutorials
ex. https://docs.chia.net/guides/cat-creation-tutorial